docs/configuration/http_conn_man/route_config/vcluster.rst

```diff
@@ -38,4 +38,10 @@ method
   *(optional, string)* Optionally specifies the HTTP method to match on. For example *GET*, *PUT*,
   etc.
 
+  Examples:
+
+    * The regex *^/rides/\d+$* matches the path */rides/0*
```

Comment:
Are the `^$` here redundant?

Reply:
Ah, yes, I had gotten this example from a test case and forgot to remove those. Will fix